Unit Tests/Tests/Networking/SDK Info/PPSDKInfoExtensionsTests.swift

```diff
@@ -0,0 +1,34 @@
+import XCTest
+import PusherPlatform
+@testable import PusherChatkit
+
+class PPSDKInfoExtensionsTests: XCTestCase {
+    
+    // MARK: - Tests
+    
+    func testShouldHaveCurrentSDKInfo() {
+        XCTAssertNotNil(PPSDKInfo.current)
+    }
+
+    // FIXME: Uncomment this test after changing access level of PPSDKInfo properties in PusherPlatform SDK.
+//    func testCurrentSDKInfoShouldHaveProductNameSetToChatkit() {
```

Comment:
Should we remove commented code out of this commit? We can add this in later when it will be needed.

Reply:
This is a valid test that should be there. In my opinion getting rid of this now would effectively mean that we will never write that unit test in future (we will have more important tasks to do).